### PR TITLE
Added advisor and meeting time/place information.

### PR DESCRIPTION
**Pull request by treyfortmuller**
v1.2.1

### CODE REVIEW

File: tecclub.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
-%YAML 1.2
+%YAML 1.2.1
```
Comment (awendland) [inline tecclub.yaml:1]: The "%YAML 1.2" denotes the version of YAML Spec that this file adheres to

File: tecclub.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 ---
 TEC Club:
     Founded: 2013-10-01
     Location: Corona del Mar HS
+    Advisor: Ms. Fletcher (afletcher@nmusd.us)
+        - 8th grade English teacher
+
+Meetings: Every Friday @ lunch, room 237
```
Comment (awendland) [inline tecclub.yaml:9]: Proper indentation is mandatory in YAML. This line needs to be indented 4 spaces

File: tecclub.yaml (hunk 1 above)
Comment (treyfortmuller) [inline tecclub.yaml:1]: ahhh... I must've gotten too excited.

File: tecclub.yaml [hunk 3]
```diff
@@ -1,6 +1,10 @@
-%YAML 1.2
+%YAML 1.2.1
 ---
 TEC Club:
     Founded: 2013-10-01
     Location: Corona del Mar HS
+    Advisor: Ms. Fletcher (afletcher@nmusd.us)
+        - 8th grade English teacher
```
Comment (awendland) [inline tecclub.yaml:7]: The "-" represents a list item in YAML. Making this a key/value pair would probably be better.
"    Email: afletcher@nmusd.us"
"    Position: 8th grade English teacher"

Comment (treyfortmuller) [inline tecclub.yaml:7]: Let it be know that I know nothing about YAML formatting, we're all learning already!

**Pull request closed without merging**